cocotb/share/lib/gpi_log/gpi_logging.cpp

```diff
@@ -185,12 +185,14 @@ static void gpi_log_v(const char *name, enum gpi_log_levels level, const char *p
         goto error;
     }
 
-    PyObject *filter_ret = PyObject_CallFunctionObjArgs(pLogFilter, logger_name_arg, level_arg, NULL);
+    PyObject *filter_ret;
+    filter_ret = PyObject_CallFunctionObjArgs(pLogFilter, logger_name_arg, level_arg, NULL);
     if (filter_ret == NULL) {
         goto error;
     }
 
-    int is_enabled = PyObject_IsTrue(filter_ret);
+    int is_enabled;
+    is_enabled = PyObject_IsTrue(filter_ret);
     Py_DECREF(filter_ret);
```

Comment:
Wrapping these lines in braces would allow you to keep `filter_ret` on a single line.

You'd need to pull `is_enabled` out of the braces, but I think it would overall add clarity